src/new/wallet.js

```diff
@@ -673,11 +673,13 @@ class HathorWallet extends EventEmitter {
       if (count <= 0) {
         break;
       }
+      const txbalance = await this.getTxBalance(tx);
       txs.push({
         txId: tx.tx_id,
         timestamp: tx.timestamp,
         tokenUid: uid,
         voided: tx.is_voided,
+        balance: txbalance[uid] || 0,
```

Comment:
Not sure if we need in one of the wallets but the wallet service facada also has the tx version in this return object. And it doesn't have the `tokenUid`. If we are using it, the wallet service won't have this information.